decidim-participatory_processes/app/cells/decidim/participatory_processes/content_blocks/democratic_quality_stats_settings_form_cell.rb

```diff
@@ -0,0 +1,119 @@
+# frozen_string_literal: true
+
+module Decidim
+  module ParticipatoryProcesses
+    module ContentBlocks
+      # Cell for the democratic quality stats settings form content block.
+      class DemocraticQualityStatsSettingsFormCell < Decidim::ContentBlocks::BaseCell
+        alias form model
+
+        def content_block
+          options[:content_block]
+        end
+
+        def i18n_scope
+          "decidim.participatory_processes.admin.content_blocks.democratic_quality_stats_settings_form"
```

Comment:
Why not to include this scope in the ignore_unused section of config/i18n-tasks.yml (adding an extra key if the same scope is being used in other parts) to avoid the comments?